recipe/meta.yaml

```diff
@@ -85,7 +85,7 @@ requirements:
   host:
     - python
     - numpy
-    - eigen
+    - eigen =3.4.0
```

Comment:
Why this change

Reply:
As it's stated in the description: `Pinned eigen dependency, because newer patch version (released today) has issues with compilation.`

Reply:
Just FYI, that version of eigen was not actually meant to be released: https://github.com/conda-forge/eigen-feedstock/issues/39 .